easybuild/easyconfigs/g/GNUCobol/gnucobol-3.0-rc1-GCC-6.2.0.eb

```
@@ -0,0 +1,27 @@
+# Lev Lafayette, University of Melbourne, 20190808
+easyblock = 'ConfigureMake'
+
+name = 'gnucobol'
+version = '3.0-rc1'
+
+homepage = 'https://sourceforge.net/projects/open-cobol/'
+description = """GnuCOBOL (formerly OpenCOBOL) is a free COBOL compiler. cobc translates COBOL source to executable using intermediate C, designated C compiler and linker."""
+
+toolchain = {'name': 'GCC', 'version': '6.2.0'}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = ['http://downloads.sourceforge.net/open-cobol/gnucobol-3.0-rc1.tar.xz']
+
+dependencies = [
+        ('DB', '6.2.32'),
+        ('GMP', '6.1.1'),
```

```suggestion
        ('GMP', '6.1.2'),
```
to match the existing version for this toolchain